test/stress_tests/run_application_stress_tests.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+# This script runs all of the application tests.
+
+
```

Comment:
Could you document usage at the top of this script? E.g., the `DEBUG_MODE` environment variable is not documented.